common/src/main/java/com/microsoft/identity/common/internal/cache/BrokerApplicationMetadata.java

```diff
@@ -41,33 +41,17 @@ public void setFoci(final String mFoci) {
         this.mFoci = mFoci;
     }
 
-    //CHECKSTYLE:OFF
-    // This method is generated. Checkstyle and/or PMD has been disabled.
-    // This method *must* be regenerated if the class' structural definition changes through the
-    // addition/subtraction of fields.
-    @SuppressWarnings("PMD")
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-        if (!super.equals(o)) return false;
-
-        BrokerApplicationMetadata metadata = (BrokerApplicationMetadata) o;
-
-        return mFoci != null ? mFoci.equals(metadata.mFoci) : metadata.mFoci == null;
+        // This class intentionally no longer includes foci state to determine equality.
```

Comment:
If we're making equals behave non-intuitively, then we need to have this called out strongly in the method and class javadoc.  I think there are several places where we've dealt with equals in interesting ways.

Reply:
2d12a83145df649da24048f059d15154b6a86b03